src/TodoInput.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const TodoInput = ({onAdd}) => {
+
+  const onKeyPress = (event) => {
+    const title = event.target.value;
+    if (event.key === 'Enter' && title.trim() !== "") {
+      onAdd(title);
+      event.target.value = "";
+    }
+  }
+
+  return (
+    <div>
+      <input
+        id="new-todo-title"
+        className="new-todo"
+        placeholder="할일을 추가해 주세요"
+        onKeyPress={onKeyPress}
```

Comment:
```suggestion
        onKeyPress={handleKeyPress}
```

와 같이 네이밍을 해주면 조금 더 가독성을 높일 수 있습니다!